packages/arb-bridge-eth/contracts/rollup/RollupCore.sol

```diff
@@ -352,6 +352,7 @@ contract RollupCore is IRollupCore {
 
     /**
      * @notice Reduce the stake of the given staker to the given target
+     * @dev Value deducted is accessible by pulling from _withdrawableFunds mapping
```

Comment:
I think the "withdrawableFunds" is self explanatory.

It might be interesting to instead mention the top level function that is callable to access your `_withdrawableFunds`. Also this is an internal function, it might be worth instead adding this comment to the top level function that calls `reduceStakeTo`

Reply:
ye- removing and adding a line to reduceDeposit (generally, wanted to make it more obvious which methods are moving funds in and out of stake and which are slashing)